files/build_templates/docker_image_ctl.j2

```diff
@@ -287,6 +288,7 @@ start() {
 {%- endif %}
     docker create {{docker_image_run_opt}} \
         --net=$NET \
+        -e RUNTIME_OWNER=local \
```

Comment:
why local here?

Reply:
still have this question.

Reply:
_Copy/paste from another place._

This is the 'who' running the container. As name suggests, runtime owner of this container instance.

In case of 'local' "docker create" in /usr/bin/.sh will create the container instance with "-e RUNTIME_OWNER=local" (changes in files/build_templates/docker_image_ctl.j2).

When started by kube, the manifest would/expected-to set the environment variable as below.
env:
- name: RUNTIME_OWNER
value: "kube"

When the container is started, it passes this RUNTIME_OWNER as the current to container_state, which it sets in current_owner in STATE_DB.

Being hardcoded in "/usr/bin/.sh", the local instance is assured to get this ENV var. So default to Kube, if in case missing.